src/canvas-extractor/canvas_extractor/mapping/section_associations.py

```diff
@@ -0,0 +1,123 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from datetime import datetime
+from pandas import DataFrame
+
+from .constants import SOURCE_SYSTEM, ACTIVE
+
+
+def _get_enrollment_status(status: str) -> str:
+    if status == "active":
```

Comment:
Curious why these are being mapped.  Did we normalize enrollment status?

Reply:
I am just copying the statusses present in Schoology, but I did not find a standard for it